source/manuals/programming-languages/editorconfig

```diff
@@ -1,15 +1,50 @@
-[*.java]
+# http://EditorConfig.org
+# https://gds-way.cloudapps.digital/manuals/programming-languages/editorconfig
+# Copy into project root and rename to .editorconfig
+
+# uncomment if top-most EditorConfig file
+# root = true
+
+[*]
```

Comment:
I think I'd prefer not to see options applied globally—I suspect it will be confusing or incorrect when applied to certain file types.

Reply:
Good point, it does act as a catch-all so may have unintended consequences